### DISCUSSION TITLE
Scrollable child

### DISCUSSION THREAD

**Discussion by jerome-pialot** [DevbookHQ/splitter#21] [Q&A] [answered]
Hello everyone,
I found the splitter component very interesting and usefull. 
Unfortunatly I encounter some troubles trying to manage scrollable child and or to get a clean child panel without overlapping.
Do you have some prerequisits to make all works correctly?
Does someone have some samples around splitter and scrollable contents?
Many thanks for your help

![image](https://user-images.githubusercontent.com/94443956/196630810-3bb22fa5-6413-4deb-accf-8817c10502e8.png)
![image](https://user-images.githubusercontent.com/94443956/196630836-f9dd7448-711a-4efc-8ac7-3d5a20044f67.png)

**Comment by jerome-pialot** [comment, marked as answer]
Solved...
I just need to use a main first unique container / div / box by splitted zone 100% width & height :-)

**Reply by mlejva** [reply]
Hey, glad you figured out!
